keps/sig-cli/3638-kubectl-plugin-subcommands/README.md

```diff
@@ -274,11 +274,12 @@ in back-to-back releases.
 
 - Gather feedback from developers and surveys
 - Add integration tests
-- Enable shadowing for all non-existent subcommands
+- Remove `KUBECTL_ENABLE_CMD_SHADOW` environment variable and this feature will be enabled by default.
```

Comment:
Should we remove it or default to `true` if it is not explicitly set to `false`? If we remove it, then this is basically stable.

Seems a little far fetched that someone might need to turn it off, but it's also not a difficult change to make.

Reply:
Thanks @johnbelamaric. I've updated the KEP by setting environment variable to true instead removing it. Let me know what you think?.